scripts/screener/screener.config.js

```diff
@@ -30,34 +30,42 @@ require('tsconfig-paths').register({
 });
 
 const baseBranch = 'master';
-const sourceBranch = process.env.BUILD_SOURCEBRANCH;
-
-// https://github.com/screener-io/screener-runner
-module.exports = {
-  apiKey: process.env.SCREENER_API_KEY,
-  projectRepo: 'microsoft/fluentui/fluentui',
-
-  diffOptions: {
-    structure: true,
-    layout: true,
-    style: true,
-    content: true,
-    minLayoutPosition: 1, // Optional threshold for Layout changes. Defaults to 4 pixels.
-    minLayoutDimension: 1, // Optional threshold for Layout changes. Defaults to 10 pixels.
-    minShiftGraphic: 1, // Optional threshold for pixel shifts in graphics.
-    compareSVGDOM: false, // Pass if SVG DOM is the same. Defaults to false.
-  },
-
-  // screenshot every example in maximized mode
-  states: require('./screener.states').default,
-
-  alwaysAcceptBaseBranch: true,
-  baseBranch,
-  failureExitCode: 0,
-
-  ...(sourceBranch && sourceBranch.indexOf('refs/pull') > -1
-    ? {
-        commit: getCurrentHash(),
-      }
-    : null),
-};
+
+/**
```

Comment:
I was confused where is this being used. after checking the source it looks like it is used only within gulp task thus for v0. can we rename this file (or move to proper boundary) and add docs  ? can be done as follow-up

Reply:
makes sense, will make a follow up PR for this